core/trino-main/src/test/java/io/trino/sql/planner/TestConnectorExpressionTranslator.java

```diff
@@ -455,6 +456,39 @@ public void testTranslateRegularExpression()
                 });
     }
 
+    @Test
+    void testTranslateJsonPath()
+    {
+        // JsonPath type is considered implementation detail of the engine
```

Comment:
`JsonPath` looks like a formal name, but the name is actually `JsonPathType`

Let's maybe write it as "JSON path type"